_i18n/en/_posts/2018-09-19-dash-core-release.md

```diff
@@ -0,0 +1,25 @@
+---
+author: tungfa
+layout: post
+image: "release-note.png"
+title: "Dash Core 0.12.3.3 Release Announcement"
+original-author: UdjinM6
+original-link: https://www.dash.org/forum/threads/dash-core-0-12-3-3-release-announcement.40698/
+---
+
+Dash Core 0.12.3.3 is a critical bugfix release of the Dash Core 0.12.3.x series.
+
+There was a serious vulnerability discovered in Bitcoin Core's codebase recently which
+can cause node receiving a specially crafted block to crash [https://github.com/bitcoin/bitcoin/pull/14247d](https://github.com/bitcoin/bitcoin/pull/14247)
+
+
+We encourage all full node operators (masternodes, mining pools, explorers, exchanges etc.) to update ASAP.
+
+Full release notes are available at [https://github.com/dashpay/dash/blob/v0.12.3.3/doc/release-notes.md](https://github.com/dashpay/dash/blob/v0.12.3.3/doc/release-notes.md),
+release binaries can be downloaded from [https://github.com/dashpay/dash/releases/tag/v0.12.3.3](https://github.com/dashpay/dash/releases/tag/v0.12.3.3).
+
+Note: this release does NOT require masternode owners to start their masternodes from cold wallets.
+Masternode operators should stop the daemon running on a remote node, replace old binaries with the new ones and start the daemon again.
+
+Same ann on github:
```

Comment:
This section should not be included imo, this was meant for info sources we do not control.